src/swap/useSwapQuote.ts

```diff
@@ -183,7 +187,7 @@ function useSwapQuote({
         return null
       }
 
-      if (!swapAmount[updatedField].gt(0)) {
+      if (!swapAmount[updatedField] || !swapAmount[updatedField].gt(0)) {
```

Comment:
i wonder if we need this condition: `!swapAmount[updatedField].gt(0)`?
i noticed that we do almost the same check against the `swpapAmountInWay` just below

Reply:
@bakoushin I've just adjusted the existing logic to reduce the amount of potential changes. But I can remove if needed!